pkg/autostart/io.lima-vm.autostart.INSTANCE.plist

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>Label</key>
+	<string>io.lima-vm.autostart.{{ .Instance }}</string>
+	<key>ProgramArguments</key>
+	<array>
+		<string>{{ .Binary }}</string>
+		<string>start</string>
+		<string>--foreground</string>
+		<string>--pidfile</string>
+		<string>ha.pid</string>
+		<string>--socket</string>
+		<string>ha.sock</string>
```

Comment:
The `limactl start` command does not have the `--pidfile` and `--socket` options.
```suggestion
```

Reply:
Thank you for the tip 🙏